applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -270,11 +301,14 @@ int bt_le_audio_tx_init(void)
 	}
 
 	for (int i = 0; i < SRC_STREAM_COUNT; i++) {
-		tx_info[i].iso_tx_pool = iso_tx_pools[i];
-		tx_info[i].hci_wrn_printed = false;
-		tx_info[i].iso_tx.ts = 0;
-		tx_info[i].iso_tx.offset = 0;
-		tx_info[i].iso_tx.seq_num = 0;
+		tx_info_arr[i].iso_tx_pool = iso_tx_pools[i];
+		tx_info_arr[i].hci_wrn_printed = false;
+		tx_info_arr[i].iso_tx.ts = 0;
+		tx_info_arr[i].iso_tx.offset = 0;
+		tx_info_arr[i].iso_tx.seq_num = 0;
+		tx_info_arr[i].iso_tx_readback.ts = 0;
+		tx_info_arr[i].iso_tx_readback.offset = 0;
+		tx_info_arr[i].iso_tx_readback.seq_num = 0;
```

Comment:
Is it possible to just use memset instead? Otherwise we might forget to reset one new addition to the struct in the future

Reply:
Possible, but it may be that not all new members should be zero.